target/linux/ath79/dts/qca9563_nec_wg800hp.dts

```diff
@@ -142,6 +142,24 @@
 					#address-cells = <1>;
 					#size-cells = <1>;
 
+					macaddr_board_data_280: macaddr@280 {
+						compatible = "mac-base";
+						reg = <0x280 0x11>;
+						#nvmem-cell-cells = <1>;
+					};
+
+					macaddr_board_data_480: macaddr@480 {
```

Comment:
where is this used?

Reply:
It’s not used yet. I mentioned in the commit message it requires DSA